spec/models/asset_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe Asset, type: :model do
+  it 'should be of class Asset' do
+    expect(subject.class).to eq Asset
+  end
+
+  describe 'Database table' do
+    it { should have_db_column :id }
+    it { should have_db_column :created_at }
+    it { should have_db_column :updated_at }
+    it { should have_db_column :file_file_name }
+    it { should have_db_column :file_content_type }
+    it { should have_db_column :file_file_size }
+    it { should have_db_column :file_updated_at }
+    it { should have_db_column :created_by_id }
+    it { should have_db_column :last_modified_by_id }
+    it { should have_db_column :estimated_size }
+    it { should have_db_column :file_present }
+    it { should have_db_column :lock }
+    it { should have_db_column :lock_ttl }
+    it { should have_db_column :version }
+    it { should have_db_column :file_processing }
+    it { should have_db_column :team_id }
```

Comment:
I mostly used shoulda matchers for associations and validations.

It is good to have expectations on columns but the down side is that the tests are fragile. If you change column name you need to always change tests.